client/components/TestPlanUpdater/TestPlanUpdaterModal.jsx

```diff
@@ -0,0 +1,541 @@
+                            </div>
+                            <div className="current-version version-info-label">
+                                <b>Current version:</b>{' '}
+                                {gitUpdatedDateToString(
+                                    currentVersion.updatedAt
+                                )}{' '}
+                                {currentVersion.gitMessage} (
+                                {currentVersion.gitSha.substring(0, 7)})
+                            </div>
+                            <div className="new-version version-info-label">
+                                <b>Latest version: </b>
+                                {gitUpdatedDateToString(
+                                    latestTestPlanVersion.updatedAt
+                                )}{' '}
+                                {latestTestPlanVersion.gitMessage} (
+                                {latestTestPlanVersion.gitSha.substring(0, 7)})
+                            </div>
+                        </div>
+                        <div>
+                            {(() => {
+                                if (!versionData) {
+                                    return (
+                                        <p>
+                                            The number of test results to copy
+                                            will be shown here after you choose
+                                            a new version.
+                                        </p>
+                                    );
+                                }
+                                if (runsWithResults.length === 0) {
+                                    return (
+                                        <p>
+                                            There are no test results associated
+                                            with this report.
+                                        </p>
+                                    );
+                                }
+
+                                const testers = runsWithResults.map(
+                                    testPlanRun => testPlanRun.tester.username
+                                );
+
+                                let deletionNote;
+                                if (!testsToDelete.length) {
+                                    deletionNote = (
+                                        <>
+                                            All test results can be copied from
+                                            the old report to the new report.
+                                        </>
+                                    );
+                                } else {
+                                    deletionNote = (
+                                        <>
+                                            Note that {testsToDelete.length}{' '}
+                                            tests differ between the old and new
```

Comment:
Could we just make it "test" in the case that there's just one test which can't be copied?